tapgarden/custodian.go

```diff
@@ -546,12 +546,15 @@ func (c *Custodian) importAddrToWallet(addr *address.AddrWithKeyInfo) error {
 	// along if so.
 	case strings.Contains(err.Error(), "already exists"):
 
-	case err != nil:
+	default:
```

Comment:
Same here, or there's a condition that `ptrAddr` is nil but the error non-nil? Either way, seems equiv.